Scripts/dat/trebivdil.fni

```diff
@@ -0,0 +1,4 @@
+Graphics.Renderer.SetClearColor 0.5 0.699999988079071 0.800000011920929
```

Comment:
Are these numbers intentionally specific, or just IEEE 754 rounding errors for 0.7 and 0.8?

Reply:
They're entered as 0.7 and 0.8 in Korman, but export that way. /shrug